### ISSUE TITLE
Improving code style

### ISSUE THREAD

**Issue by Superfranci99** [TeamFail/Trajectory#1]
I advise you to start coding with OOP paradigms in the way to perform your algorithms. That is I guess you should start working with classeses and not writing everything in the form1.cs class.
Unfortunately at the moment I haven't got a lot of free time to help you. When I'll get some free time I will be pleased to help you improving the code style

Francesco

**Comment by TeamFail** [comment]
Yeah, I wasn't expecting to release this, let alone expect the mess that the form1.cs class was going to become. I do intend to rewrite it once I finish up with a few things though, I do have a few ideas that I want to implement.

**Comment by Superfranci99** [comment]
Yeah I understand your position. I advise too another way to write better code: TDD. At the beginning it isn't very simple to understand, but after your code will be stronger and safer.

Another thing that I think should be really important is looking the arm code of the game, how it actually reads all the informations from the file structure. Most of the times you will find some interesting things and jokes of the nds sdk... You may discover unused variables or maybe some other that are used for other thing you couldn't absolutely imagine.
